apps/passport-client/components/screens/FrogScreens/ManageFeedsSection.tsx

```diff
@@ -302,15 +304,15 @@ export function DataTable({
       noDataMessage="No Feeds"
       customRenderCell={{
         ...keys.reduce((acc, key) => {
-          acc[key] = (row) => {
+          acc[key] = (row): any => {
```

Comment:
Is `any` the right thing here?

Reply:
In this specific case, we are not getting good type info out of `react-table-lite`. The types here are the best that TypeScript is able to infer. If I knew the library well I might be able to specify it more tightly.